### ISSUE TITLE
how to import into eclipse

### ISSUE THREAD

**Issue by drandreaskrueger** [webdaford/smart-contract-tools#27]
1. Please provide a step by step manual.  
2. Even better: Include into Eclipse marketplace.

But 1 would help already. 

Thanks.

**Comment by drandreaskrueger** [comment]
In Eclipse Preferences "available software sites", I seem to be able to input a `http:` or a `jar:` resource.

Would it help, and be possible that you provide a jar file then?

**Comment by drandreaskrueger** [comment]
I have just found this in the marketplace:

> Update site URL×
> Add this URL to your Eclipse Installation to reach this Solution's Update Site.
> Mars (4.5)
> http://urszeidler.github.io/uml2solidity/uml2solidity-updatesite/

But then, in eclipse:

> Cannot perform operation. Computing alternate solutions, may take a while: 7/15

etc.

Fails.

Edit: I found the author, and created an [issue there](https://github.com/UrsZeidler/uml2solidity/issues/11).

But still, your repo here needs a howto.md, no?

**Comment by UrsZeidler** [comment]
Yeah, there is the the issue #7 to provide an update site, Dan is quite busy, when you have maven at you disposal  you could simply build it on your own.

**Comment by drandreaskrueger** [comment]
> when you have maven at you disposal

Sounds a bit as if that (maven) is again a few hours to get it all started. So let's say: No. 
And I just discovered that Javascript syntax highlighting does a mildly good job already.

> to provide an update site

Why site? Is there no way to import it as a simple file archive?

And I am actually surprised that there is little support for Eclipse here. Am I from another generation? ;-) Is it time to be dropping Eclipse perhaps?

Anyways, thanks!

**Comment by UrsZeidler** [comment]
I also used the javascript editor before that. ;)

For you I have put my private build of the editor here:
http://opensource.urszeidler.de/SolidityEditor-1.0.0-SNAPSHOT.zip

If you like try this.
It's an ziped updatesite and should work well. In this private build are some feature added (for example proposals) which are not included here right now.

**Comment by drandreaskrueger** [comment]
> For you I have put my private build of the editor here:

oh thank you very much!

I suppose it was correct to choose "content.jar" for installing? Or rather/first "artifacts.jar"?

content.jar -->

```
An error occurred while collecting items to be installed
session context was:(profile=nodeclipse.package.enide.product, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
No repository found containing: osgi.bundle,com.dell.research.bc.eth.solidity.editor,1.0.0.201605301655
No repository found containing: org.eclipse.update.feature,com.dell.research.bc.eth.solidity.editor.feature,1.0.0.201605301655
No repository found containing: osgi.bundle,com.dell.research.bc.eth.solidity.editor.ui,1.0.0.201605241722
No repository found containing: osgi.bundle,com.dell.research.bc.eth.solidity.ui,1.0.0.201512170225
No repository found containing: osgi.bundle,com.dell.research.bc.eth.ui,1.0.0.201512170225
```

**Comment by UrsZeidler** [comment]
No.
The zip contains complete the update site.
To install you need to choose help->install new software.
Select 'add' for a new update site and choose archive, and as archive choose the zip file.

**Comment by drandreaskrueger** [comment]
Ah :-) sorry, I was mislead by the `jar:` input in eclipse.

Very good, it works. SUPERB, I have a solidity editor now. (Now the contract reading can begin. I want to study good examples first. My first choice was 'wallet.sol', and already learned a good bit there.)

Thanks a lot.  Putting this valuable ZIP into the README.md I suggest.

**Comment by webdaford** [comment]
Thanks for the comments and suggestions.  I've been occupied by a few other things.  Looking at it now.

**Comment by drandreaskrueger** [comment]
Thx @webdaford . For me the issue is solved since I could install @UrsZeidler 's zip archive. 

IMHO would make sense for you to officially provide exactly that.

Thanks for your great tool!
